app/src/test/java/io/crate/plugin/PluginLoaderTest.java

```diff
@@ -54,29 +54,6 @@
         return Arrays.asList(PluginLoaderPlugin.class, Netty4Plugin.class);
     }
 
-    @Test
-    public void testLoadPlugin() throws Exception {
```

Comment:
Plugin providing scalars would not work anymore, as they rely on the old function registry including old methods.
Is it worth to keep compatibility given that our plugin infrastructure seems to be unused?

Reply:
> s it worth to keep compatibility given that our plugin infrastructure 

No I don't think so.
If people request this we can either create a narrow (Scalar)FunctionPlugin or extend the user-defined-functions to support Java as well - which would also avoid the deployment hassle